ConfigSpace/hyperparameters.pyx

```diff
@@ -1542,84 +1553,93 @@ cdef class UniformIntegerHyperparameter(IntegerHyperparameter):
 
     def get_neighbors(
         self,
-        value: Union[int, float],
+        value: float,
         rs: np.random.RandomState,
         number: int = 4,
         transform: bool = False,
         std: float = 0.2,
     ) -> List[int]:
-        cdef int n_requested = number
-        cdef int idx = 0
-        cdef int i = 0
-        neighbors = []  # type: List[int]
-        cdef int sampled_neighbors = 0
-        _neighbors_as_int = set()  # type: Set[int]
-        cdef long long int_value = self._transform(value)
-        cdef long long new_int_value = 0
-        cdef float new_value = 0.0
-        cdef np.ndarray samples
-        cdef double[:] samples_view
-
-        if self.upper - self.lower <= n_requested:
```

Comment:
Do you think keeping this could reduce the runtime in edge cases? I would bet we can reduce it quite a bit again by re-using this little for-loop

Reply:
Put it back in :)